src/api/lib/tasks/dev.rake

```diff
@@ -249,6 +249,65 @@ namespace :dev do
     end
   end
 
+  # Run this task with: rails dev:workflow_runs:data"
+  namespace :workflow_runs do
+    desc 'Creates workflow runs, artifacts and all its dependencies'
+    task data: :environment do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
```

Comment:
soo why isn't all of this a factory?

Reply:
I added a new commit on top to create the objects to which the artifacts link to.